tests/Sentry/Metrics/Listener/OnBeforeHandleTest.php

```
@@ -0,0 +1,67 @@
+use FriendsOfHyperf\Sentry\Feature;
+use FriendsOfHyperf\Sentry\Metrics\Listener\OnBeforeHandle;
+use Hyperf\Command\Command;
+use Hyperf\Command\Event\BeforeHandle;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Console\Application;
+
+beforeEach(function () {
+    Constants::$runningInCommand = false;
+
+    $this->timer = new FakeTimer();
+    $this->container = $this->createMock(ContainerInterface::class);
```

PHPStan (2.2.7)

[error] Call to protected method createMock() of class PHPUnit\Framework\TestCase.


(method.protected)